### DISCUSSION TITLE
How do I get someone to look at my [BUG] issue?

### DISCUSSION THREAD

**Discussion by JazBInKC** [pnp/powershell#1806] [Q&A]
The title says it all.  Several weeks since I submitted the issue and recommended a possible remedy.  Maybe I don't understand the review process, but it would be nice to have someone take a look.

https://github.com/pnp/powershell/issues/1716

Thanks!

jbw

**Comment by AndersRask** [comment]
Please consider that the PnP.PowerShell framework is driven by people who do it on their own time, as this is not an official Microsoft framework.
If you think you have a solution, you are free to download repository and fix it, check if it works better and then submit a pull request describing the changes.
Also consider that a memory leak might also be from the PnP.Core framework underneath this framework.

**Reply by sparkitect** [reply]
As well: Some things are broken at a level deeper than PnP.

### Private Channel sites are FAST to create, but still borked https://github.com/pnp/powershell/discussions/1770

**Comment by sparkitect** [comment]
@JazBInKC 

Timing is of essence: Currently, those that are best suited to respond are all gathered at/in a conference or community call or are at work right now.

And also right now.

Attention is no longer a commodity; some demands for it require patience while others require hunting and gathering ;)
